specification/translation/Azure.AI.DocumentTranslation/routes.tsp

```diff
@@ -421,6 +423,6 @@ interface DocumentTranslationOperations {
       @query("type")
       type?: FileFormatType;
     },
-    SupportedFileFormats
+    FileFormat[]
```

Comment:
Why are we changing this return type? Was it incorrect before?

Reply:
The SupportedFileFormats type will not contain anything else except the List, hence Srikanta had suggested to just return FileFormat[]

Reply:
Can you show me an example of what the service response looks like for this endpoint? Because what was specified before is not the same as what we're specifying now. SupportedFileFormats had a value property that was returning the FileFormat array, now you're saying that the response is the array directly?

Reply:
You are right, we do return a value property with an array of FileFormats. 
Reverted this change, thank you for pointing this out.
Here is the service response:
{
    "value": [
        {
            "format": "XLIFF",
            "fileExtensions": [
                ".xlf"
            ],
            "contentTypes": [
                "application/xliff+xml"
            ],
            "defaultVersion": "1.2",
            "versions": [
                "1.0",
                "1.1",
                "1.2"
            ],....